docs/fundamentals/read-operations.txt

```diff
@@ -269,6 +271,103 @@ documents.
             Plot: A sci-fi update of the famous 6th Century poem. In a beseiged land, Beowulf must
             battle against the hideous creature Grendel and his vengeance seeking mother.
 
+.. _laravel-sort:
+
+Sort Query Results
+~~~~~~~~~~~~~~~~~~
+
+To order query results based on the values of specified fields, use the ``where()`` method
+followed by the ``orderBy()`` method. You can set an **ascending** or
+**descending** sort direction on results. By default, the ``orderBy()``
+method sets an ascending sort on the supplied field name. To specify a
+descending sort, pass ``'desc'`` as the second parameter.
```

Comment:
Suggestion:
I think the orderBy() method also accepts `asc' for ascending sort. It could be helpful to let the know the user exists (for completeness) and to mention that it's not necessary since it's the default value. I think keeping the asc examples as-is is fine though.